src/style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Book.ttf) format("truetype");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Heavy.ttf) format("truetype");
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  margin-top: 20px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.icon::placeholder {
+  font-family: inherit;
+  font-weight: 300;
+}
+
+.bigIcon {
+  font-family: "Avenir", georgia, serif;
+  font-size: 16px;
+  vertical-align: top;
+  height: 70px;
+  padding: 0 0 0 62px;
+  width: 100%;
+  margin-bottom: 20px;
+  outline: none;
```

Comment:
you shouldn't duplicate styles in the big input and small input. Write common styles in common class (checklist 2)